src/services/BooksService.ts

```diff
@@ -0,0 +1,17 @@
+import api from '~config/api';
+import { CLIENT_KEY, TOKEN_KEY, UID_KEY } from '~utils/constants';
+import { BooksRequestSucess, BooksRequestFail } from '~utils/types';
+
+import LocalStorageService from './LocalStorageService';
+
+const GET_BOOKS_URL = 'books';
+const HEADERS = {
+  [TOKEN_KEY]: LocalStorageService.getValue(TOKEN_KEY),
+  client: LocalStorageService.getValue(CLIENT_KEY),
+  uid: LocalStorageService.getValue(UID_KEY)
+};
+
+export const getBooks = () => {
+  api.setHeaders(HEADERS);
```

Comment:
this should be set on login, and when you refresh or open the app, you should take this from local storage and set the api headers, remember that this headers when the user is logged in, should always be set only once to every request